_data/events.yml

```diff
@@ -1,19 +1,14 @@
-  - date: 2016-09-16
-    name: Strange Loop
+  - date: 2016-11-30
+    name: AWS re:Invent
     info: "Druid: Powering Interactive Data Applications at Scale"
```

Comment:
Is this "info" right? The linked page says the talk is titled "Disrupting Big Data with Cost-effective Compute"